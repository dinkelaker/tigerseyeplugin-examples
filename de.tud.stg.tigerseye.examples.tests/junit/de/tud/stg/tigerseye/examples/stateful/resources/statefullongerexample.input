	set "OS" = "unix"	
	set "isValid" = true
	
	set "key1" = "val1"
	println get"key1"
	set"key2"="val2"
	println get"key2"
	set"key3" =                  
	
	
	
	"val3"
	println get"key3"
	
	println "isValid is:" + get "isValid"
	
	if("notValid") set "isValid" = false
	
	println "isValid is:" + get "isValid"
		
	println "OS1 is: " + get "OS"
	
	println "OS2 is: " + get"OS"
	
	println "OS3 is: " + get           
	
	
	
	
	
	
	 "OS";