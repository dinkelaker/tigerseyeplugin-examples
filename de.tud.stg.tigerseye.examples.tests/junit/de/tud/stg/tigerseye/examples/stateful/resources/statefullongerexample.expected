set__p0__equals__p1(
"OS",
"unix")	
	set__p0__equals__p1(
"isValid",
true)
	
	set__p0__equals__p1(
"key1",
"val1")
	println get__p0(
"key1")
	set__p0__equals__p1(
"key2",
"val2")
	println get__p0(
"key2")
	set__p0__equals__p1(
"key3",
"val3")
	println get__p0(
"key3")
	
	println "isValid is:" + get__p0(
"isValid")
	
	if("notValid") set__p0__equals__p1(
"isValid",
false)
	
	println "isValid is:" + get__p0(
"isValid")
		
	println "OS1 is: " + get__p0(
"OS")
	
	println "OS2 is: " + get__p0(
"OS")
	
	println "OS3 is: " + get__p0(
"OS");